lnst/Tests/XDPBench.py

```diff
@@ -0,0 +1,139 @@
+import re
+import time
+import signal
+import logging
+from subprocess import Popen, PIPE
+from lnst.Devices.Device import Device
+
+from lnst.Tests.BaseTestModule import BaseTestModule, TestModuleError
+from lnst.Common.Parameters import (
+    ChoiceParam,
+    StrParam,
+    IntParam,
+    DeviceParam,
+)
+
+
+class XDPBenchOutputParser:
+    def __init__(self, process: Popen, output: list) -> None:
+        self._process = process
+        self._out = output
+
+    def parse_output(self):
+        stdout, stderr = self._process.communicate()
+
+        logging.debug("Stderr of xdp-bench:")
+        logging.debug(str(stderr))
+
+        self._parse_raw_results(stdout)
+
+        if not self._out:
+            raise TestModuleError(f"Could not parse xdp-bench output: {stdout}")
+
+    def _parse_raw_results(self, raw_results: bytes):
+        decoded = raw_results.decode()
+        intervals = decoded.splitlines()
+
+        for line in intervals:
+            self._parse_line(line)
+
+    def _parse_line(self, line: str):
+        match = re.search(r"Summary\s+([\d,]+)\srx/s\s+([\d,]+)\serr/s?", line)
+
+        if not match:  # skip summary line at the end + corrupted lines
+            logging.error(f"Could not parse xdp-bench output: {line}")
+            return
+
+        rx = match.group(1).replace(",", "")
+        err = match.group(2).replace(",", "")
+        # ^^ remove thousands separators
```

Comment:
Can't this be handled in the regex somehow?

Reply:
Nope, regex cannot ignore characters within matching group